thundra/application/global_application_info_provider.py

```diff
@@ -39,3 +42,12 @@ def get_application_info_from_config():
     def update(self, opts):
         filtered_opts = {k: v for k, v in opts.items() if v is not None}
         self.application_info.update(filtered_opts)
+
+        if not self.application_info.get('applicationInstanceId'):
+            self.application_info['applicationInstanceId'] = str(uuid.uuid4())
+
+        if not self.application_info.get('applicationId'):
+            default_app_id = 'python:{}:{}:{}'.format(self.application_info.get('applicationClassName'),
```

Comment:
if applicationClassName is `None`, how does it look like in the formatted `default_app_id`?

Reply:
Seems that it is shown as `None`. I think it is better to have it as empty string in such case